tests/init.go

```diff
@@ -232,6 +232,29 @@ var Forks = map[string]*params.ChainConfig{
 		BernoulliBlock:      big.NewInt(0),
 		CurieBlock:          big.NewInt(0),
 	},
+	"EuclidV2": {
```

Comment:
Do we need Darwin, DarwinV2, Euclid here?

Reply:
We don't need to add other hardfork configs here. because this is used for `cd ${PWD}/cmd/evm; go test -v -run=^TestT8n`, otherwise it will fail with:
```
    t8n_test.go:228: args: t8n --output.body "" --output.result stdout --output.alloc stdout --input.alloc ./testdata/33/alloc.json --input.txs ./testdata/33/txs.json --input.env ./testdata/33/env.json --state.fork EuclidV2
    test_cmd.go:281: (stderr:13) ERROR(3): failed constructing chain configuration: unsupported fork "EuclidV2"
    t8n_test.go:240: test 12, json parsing failed: unexpected end of JSON input
--- FAIL: TestT8n (0.46s)
```

Reply:
The t8n test uses this function: https://github.com/scroll-tech/go-ethereum/blob/9e70b14dec8d1386d85d590c76cbb5cc0778ced0/tests/state_test_util.go#L149-L173 to get chain config by "string" names. thus should add a map item to acknowledge the "EuclidV2" config.

The specific line of the test is here: https://github.com/scroll-tech/go-ethereum/blob/9e70b14dec8d1386d85d590c76cbb5cc0778ced0/cmd/evm/t8n_test.go#L210 (the "EuclidV2" in this line).